index.html

```diff
@@ -5,5 +5,6 @@
     </head>
     <body>
         <h1>MEOW</h1>
+        <h2><strong>Background</strong></h2>
```

Comment:
good

        <h2><strong>Background</strong></h2>
```
```suggestion
        <h2><strong>Background</strong></h2>
```
```suggestion
        <h2><strong>Background</strong></h2>
```
```suggestion
        <h2><strong>Background</strong></h2>
```
```suggestion
        <h2><strong>Background</strong></h2>
```
```suggestion
        <h2><strong>Background</strong></h2>
```